pkg/controller/sync/controller.go

```diff
@@ -807,3 +888,16 @@ func computePlacement(adapter federatedtypes.PlacementAdapter, placement pkgrunt
 	}
 	return clusterSet.Intersection(selectedClusterSet).List(), clusterSet.Difference(selectedClusterSet).List()
 }
+
+func isNamespaceKind(kind string) bool {
+	return kind == federatedtypes.NamespaceKind
+}
+
+func isSystemNamespace(namespace string) bool {
+	switch namespace {
+	case "kube-system", util.FederationSystemNamespace:
```

Comment:
Consider externalizing this list to a package var somewhere.  It will likely need to be configurable.